pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -1105,6 +1105,22 @@ func setUpstreamProtocol(cluster *apiv2.Cluster, port *model.Port) {
 			},
 		}
```

Comment:
return

Reply:
We don't need return here as both `util.IsProtocolSniffingEnabledForInboundPort(node, port)` and `util.IsProtocolSniffingEnabledForOutboundPort(node, port)` will return false.